src/libstd/macros.rs

```diff
@@ -64,6 +64,9 @@ macro_rules! panic {
 ///
 /// Equivalent to the `println!` macro except that a newline is not printed at
 /// the end of the message.
+///
+/// Note that because stdout is by default buffered on unix systems, this may
+/// result in output not being emitted until stdout is flushed.
```

Comment:
Totally reasonable, is there markup to make it actually linkify `io::stdout` ? (Will it do this by default? That would be amazing)